datasets/dataset_titanic.py

```diff
@@ -74,12 +78,15 @@ def generate_new_model_for_dataset():
     model.add(Dense(16, activation='relu'))
     model.add(Dense(8, activation='relu'))
     model.add(Dense(1, activation='sigmoid'))
-
     model.compile(optimizer = 'adam',
                    loss = 'binary_crossentropy',
                    metrics = ['accuracy'])
+    """
```

Comment:
I would remove the part where we build the neural network model since we don't use it